lib/mailersend/email/email.rb

```diff
@@ -112,6 +112,8 @@ def send
       response = client.http.post("#{API_URL}/email", json: message.delete_if { |_, value| value.to_s.strip == '' || value == [] || value == {} })
       puts response
       puts response.status.code
+      # Return the actual response here for users to perform in-app error handling
+      return response
     end
   end
 end
```

Comment:
This is out of scope of this PR - please remove it.

Reply:
Completed! Sorry; clearly this got carried over from the other changes I made on another PR lol

Reply:
No worries, we will also take review your other PR real soon!